### PR TITLE
chore: autodelete old models

### PR DESCRIPTION
**Pull request by niehaus59**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

Delete old MAD models and Face person groups in tests
## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

File: cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala [hunk 1]
```diff
@@ -217,6 +218,8 @@ class IdentifyFacesSuite extends TransformerFuzzing[IdentifyFaces] with Cognitiv
     .collect()
     .map(r => r.getString(0)).toSeq
 
+  IdentifyFacesSuite.cleanOldGroups()
+
```
Comment (mhamilton723) [inline cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala:222]: No need to make a companion object, and please call this in afterAll

File: cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala [hunk 2]
```diff
@@ -59,6 +61,8 @@ trait MADTestUtils extends TestBase with AnomalyKey with StorageCredentials {
 
 class FitMultivariateAnomalySuite extends EstimatorFuzzing[FitMultivariateAnomaly] with MADTestUtils {
 
+  cleanOldModels()
+
```
Comment (mhamilton723) [inline cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala:65]: please call this in afterAll

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Just 2 minor changes!

File: cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala (hunk 1 above)
Comment (niehaus59) [reply cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala]: Done

File: cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala (hunk 2 above)
Comment (niehaus59) [reply cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala]: Done

**Review by mhamilton723 — APPROVED** [review]

File: cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala [hunk 3]
```diff
@@ -265,17 +270,35 @@ class IdentifyFacesSuite extends TransformerFuzzing[IdentifyFaces] with Cognitiv
...
 
-  // Utility entrypoint, remove ignore to clean up manually
-  ignore("Clean up all personGroups") {
+  def cleanOldGroups(): Unit = {
+    val twoDaysAgo = LocalDateTime.now().minusDays(2)
+    PersonGroup.list().foreach { pgi =>
+      try {
+        val pgDateString = pgi.personGroupId.replaceFirst("group", "")
+        val pgDate = LocalDateTime.parse(pgDateString, DateTimeFormatter.ofPattern(timeFormat))
+        if (twoDaysAgo.compareTo(pgDate) <= 0) {
```
Comment (svotaw) [inline cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala:281]: I htink you can use something a little more readable: pgDate.isBefore(twoDaysAgo)

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**